app/templates/admin/administer-user-modal.jade

```diff
@@ -18,6 +18,7 @@ block modal-header-content
         button#deteacher-btn.btn.btn-burgundy Deteacher
   p
     button#reset-progress-btn.btn.btn-burgundy Reset Progress
+    a#export-data-btn.btn.btn-primary(target='_blank' href='/db/user/' + view.user.id + '/export-data') Export GDPR User Data
```

Comment:
Should we not use the `click` event handler instead of calling the API directly from `href` attribute just like other buttons to make it consistent?

Reply:
Nah, I had it like that but I want this to open in a new page.